server/game/cards/09.3-JfS/DojiHotaru2.js

```diff
@@ -0,0 +1,34 @@
+const DrawCard = require('../../drawcard.js');
+const AbilityDsl = require('../../abilitydsl');
+
+class DojiHotaru2 extends DrawCard {
+    setupCardAbilities() {
+        this.persistentEffect({
+            effect: AbilityDsl.effects.delayedEffect({
+                condition:  context => context.player && !!context.player.cardsInPlay.includes(card => card.name === 'Doji Kuwanan'),
+                message: '{1} is discarded from play as its controller controls {0}',
+                messageArgs: context => [context.source, context.player.cardsInPlay.find(card => card.name === 'Doji Kuwanan')],
+                gameAction: AbilityDsl.actions.discardFromPlay(context => ({
+                    target: context.player.cardsInPlay.find(card => card.name === 'Doji Kuwanan')
+                }))
+            })
+        });
+        this.reaction({
+            title: 'Gain 1 honor',
+            when: {
+                onCardPlayed: (event, context) => {
+                    return context.game.isDuringConflict() &&
+                        context.source.isParticipating() &&
```

Comment:
you don't need `isDuringConflict` if you're checking `isParticipating`

Reply:
Ok, fixed it.